pkg/cgu/cgulist_test.go

```diff
@@ -0,0 +1,58 @@
+package cgu
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	"github.com/stretchr/testify/assert"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestCguListInAllNamespaces(t *testing.T) {
+	testCases := []struct {
+		testCGU       []*CguBuilder
+		listOptions   []metav1.ListOptions
+		expectedError error
+		client        bool
+	}{
+		{
```

Comment:
Do we want to provide default values explicitly (for example listOptions here?

Reply:
Agreed